README.md

```diff
@@ -47,6 +47,36 @@ After that, you may run the binary as any normal program:
 ```
 ./shark-clipper-linux-x64-latest.bin
 ```
+#### Using the Mac Prebuilt Binary
+
+Mac requires a additional security steps to run the prebuilt binary.
+These steps need to be done every time a new binary is downloaded and run for the first time.
```

Comment:
In the preamble, mention that it is best to run these steps from a terminal.
Maybe also either link to an Apple article about the terminal, or give very simple instructions on how to open one.

On Windows we got lucky and double clicking runs it in a terminal.
Check to see if double clicking works here as well.

Reply:
Unfortunately no double click. I'm attaching some brief terminal instructions.